build/pack.c

```diff
@@ -322,9 +322,13 @@ static char *getIOFlags(Package pkg)
 static void finalizeDeps(Package pkg)
 {
     /* check if the package has a dependency with a '~' */
-    if (haveTildeDep(pkg))
+    if (haveCharInDep(pkg, '~'))
 	(void) rpmlibNeedsFeature(pkg, "TildeInVersions", "4.10.0-1");
 
+    /* check if the package has a dependency with a '^' */
+    if (haveCharInDep(pkg, '^'))
+	(void) rpmlibNeedsFeature(pkg, "CaretInVersions", "4.14.0-1");
```

Comment:
not sure if you would like me to change this to `4.15.0`

Reply:
The version in rpmlib() doesn't matter (consider backports) as long as we provide the same thing, technically it could just as well be 1.2.3.